pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -827,5 +827,20 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
           "Time column names are different for table: %s! Offline time column name: %s. Realtime time column name: %s",
           rawTableName, offlineTimeColumnName, realtimeTimeColumnName));
     }
+    TenantConfig offlineTenantConfig = offlineTableConfig.getTenantConfig();
+    TenantConfig realtimeTenantConfig = realtimeTableConfig.getTenantConfig();
+    String offlineBroker = offlineTenantConfig.getBroker();
+    String realtimeBroker = realtimeTenantConfig.getBroker();
+    if (offlineBroker == null || realtimeBroker == null) {
```

Comment:
add a unit test for this check

Reply:
@yupeng9 thanks for review! i will add a unit test after fixing the existing test failures.